components/organisms/ContentPreview.tsx

```diff
@@ -207,35 +206,42 @@ export default function ContentPreview({
       </Box>
       <DescriptionList
         nowrap
-        sx={{ mx: 2, my: 1 }}
+        sx={{ mx: 2, mt: 1 }}
         value={[
           {
             key: "更新日",
             value: getLocaleDateString(content.updatedAt, "ja"),
           },
           ...authors(content),
-          ...(content.type === "book" && content.ltiResourceLinks.length > 0
-            ? [
-                {
-                  key: "コース",
-                  value: (
-                    <Fragment>
-                      {content.ltiResourceLinks.map(
-                        (ltiResourceLink, index) => (
-                          <CourseChip
-                            key={index}
-                            ltiResourceLink={ltiResourceLink}
-                            onLtiResourceLinkClick={onLtiContextClick}
-                          />
-                        )
-                      )}
-                    </Fragment>
-                  ),
-                },
-              ]
-            : []),
         ]}
       />
+      {content.type === "book" && content.ltiResourceLinks.length > 0 && (
+        <DescriptionList
+          sx={{
+            mx: 2,
+            mb: 1,
+            dt: { flexShrink: 0 },
+            dd: { overflow: "hidden" },
```

Comment:
深い階層に対するスタイルの上書きが生じており、あまり望ましくなく感じます
本当はDescriptionListのコンポーネントの整理をおこなうべきで、例えばDescriptionItemのようなコンポーネントと分割しレイアウトの柔軟性をもたせた方がいいのかもしれません

Reply:
対応するか否か、またもし仮に対応するならば本件に含めるか否か、判断してもらえますか?

Reply:
対応した場合、今度はJSXでのテンプレートでの呼び出しの際に冗長な書き方になるような新たな問題を生む気もしまして、今の時点で対応することが良いかというと疑問が残ります
ですので @kou029w の意見等なければ対応しない方針で希望します

コードコメントとして残すことによってこの箇所について resolve にできればと思いました

```suggestion
            // TODO: 深い階層に対するスタイルの上書きが不要なコンポーネントの整理
            dt: { flexShrink: 0 },
            dd: { overflow: "hidden" },
```